lib/src/model/engine/evaluation_mixin.dart

```diff
@@ -128,6 +128,13 @@ mixin EngineEvaluationMixin {
     }
   }
 
+  @mustCallSuper
+  Future<void> stopEval() async {
+    if (evaluationState.isEngineAvailable(evaluationPrefs)) {
+      toggleEngine();
```

Comment:
We don't want toggleEngine() here. It will effectively stop the engine if it is running, but also kill it.

Here we only want to stop the running eval, not kill the engine.

You want to call `EvaluationService.stop()` instead.

Reply:
Oh that broke a test now, will have a look later

Reply:
Hmm I couldn't really figure out why exactly it broke. I went back to the original version now, but calling `requestEval()` only when the move is correct, to go back to the original user-configured search time. If the move was incorrect, we restart the eval anyway after taking it back